### DISCUSSION TITLE
Running speech on long sentences.

### DISCUSSION THREAD

**Discussion by vijayendra-tripathi** [coqui-ai/TTS#1145] [General Q&A]
I recently tried TTS and results are very exciting. However I noticed that speech breaks if a sentence gets longer. The terminal shows message **Decoder stopped with `max_decoder_steps` 500** Is there a way I can make the model run on longer sentences without retraining the model.

**Comment by Vagif12** [comment]
I'm also trying to figure this out

**Comment by erogol** [comment]
you can change `max_decoder_steps`in the model config which is where the model is downloaded in your system.

**Reply by Vagif12** [reply]
This is it, works fine now!

**Comment by Vagif12** [comment]
@erogol in the sample demo(https://erogol.github.io/ddc-samples/) for the sentence:

> Bill got in the habit of asking himself “Is that thought true?” and if he wasn’t absolutely certain it was he just let it go.

The word` wasn't `is pronounced correctly.
However, after modifying the `max_decoder_steps` in the config file of the tacotron config file, and using the above sentence with the `tts` command like so:
`tts --text "Bill got in the habit of asking himself “Is that thought true?” and if he wasn’t absolutely certain it was he just let it go."`

The word 'wasn't' is not pronounced correctly, and the intonation is different to the one seen on the demo. Any reason for this behaviour?
Thanks!

P.S - when given 2 sentences with full stops, there is a "gap" between the end of the first sentence and the beginning of second sentence. Any idea on how to solve this?
